.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,118 @@
+        - OS: [e.g. Ubuntu 22.04, macOS 13.0]
+        - Go Version: [e.g. 1.22.0]
+        - Docker Version: [e.g. 24.0.0]
+        - Database: [e.g. PostgreSQL 15.0]
+        - Redis: [e.g. Redis 7.0]
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional Context
+      description: Add any other context about the problem here
+      placeholder: Any additional information...
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)